### PR TITLE
࿓❯ fix: Removed Docker information from README

### PR DESCRIPTION
**Pull request by ZendaiOwl**
Removed the Docker information from the README

Signed-off-by: Victor-ray, S <owl@zendai.net.eu.org>

### CODE REVIEW

**Comment by theCalcaholic** [comment]
Thanks for the PR!

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This is the build code for the [NextcloudPi][ncp-website] open-source community
 
 NextcloudPi is a ready to use image for Virtual Machines, Raspberry Pi, Odroid HC1, Rock64 and other boards. ([⇒ Downloads][ncp-releases])
 
-This code also generates the NextcloudPi [Docker image][ncp-docker-hub], LXD container & VM, there is an install script for the latest stable Debian based system as well.
+<!-- This code also generates the NextcloudPi [Docker image][ncp-docker-hub], LXD container & VM, there is an install script for the latest stable Debian based system as well. -->
```
Comment (theCalcaholic) [inline README.md:22]: Can we reword this instead to

```diff
- This code also generates the NextcloudPi [Docker image][ncp-docker-hub], LXD container & VM, there is an install script for the latest stable Debian based system as well.
+ This code also generates the NextcloudPi LXD and LXC containers and there is an install script for the latest supported Debian based system as well.
```

**Review by theCalcaholic — CHANGES_REQUESTED** [review]
Other than the mentioned minor change, this looks good to me

File: README.md (hunk 1 above)
Comment (ZendaiOwl) [reply README.md:22]: Of course, updating the PR now

**Comment by m-breitbach** [comment]
Can we get this thing through to avoid more people from setting up their instances via Docker and than learn about the deprecation afterwards?

**Comment by ZendaiOwl** [comment]
@theCalcaholic could you push this into master please? :)

**Comment by theCalcaholic** [comment]
@ZendaiOwl Will do!

**Pull request merged**